test/setup/main.tf

```diff
@@ -25,9 +25,18 @@ module "project" {
   billing_account   = var.billing_account
 
   activate_apis = [
-    "healthcare.googleapis.com",
     "iam.googleapis.com",
     "pubsub.googleapis.com",
     "serviceusage.googleapis.com",
+    "cloudresourcemanager.googleapis.com",
+    "bigquery.googleapis.com"
   ]
+
+  activate_api_identities = [{
```

Comment:
why do we need this?

Reply:
Healthcare api needs these roles in order to stream to BQ (which was added to the test example).

Reply:
The healthcare SA needs this in order to enable BQ streaming.